### ISSUE TITLE
Sending link to webpage results in error message

### ISSUE THREAD

**Issue by JuliusPC** [ChatSecure/ChatSecure-iOS#1013]
Sending a [certain link](https://www.mw.niedersachsen.de/startseite/digitalisierung/buergerbeteiligung_mobilfunk/funkloch-informieren-sie-uns-ueber-mobilfunk-probleme-163504.html) to webpage results in error message (sending other links, messages and images via OMEMO is working):  
![message-error](https://user-images.githubusercontent.com/15018932/39971594-2f8d0720-56fe-11e8-92e4-27311b444674.png)

Link appears in contact sidebar:  
![conversations-list](https://user-images.githubusercontent.com/15018932/39971609-765c6df8-56fe-11e8-8e73-33ad2070a360.png)

Link sent via [Conversations](https://conversations.im/), server is trashserver.net and ChatSecure version is 4.3.0 (127). Happens on iPad and iPhone.

**Comment by Echolon** [comment]
Yep, just happened, recently with this for example to. Nevermind the content^^

https://goo.gl/images/87RSQj

**Comment by pgeorgi** [comment]
same error with https://hostname/path, works with just hostname/path. chatsecure is a bit overeager when trying to download links (which is also a privacy concern besides this usability problem)
